pages/agent/v3/securing.md.erb

```diff
@@ -30,6 +30,8 @@ By default the agent allows you to run any command on the build server (for exam
 
 Once disabled your build steps will need to be checked into your repository as scripts, and the only way to pass arguments is using environment variables.
 
+This option is intended to protect your infrastructure from a general Buildkite compromise that might attempt to send arbitrary commands to your agents. It is not designed, nor effective, at protecting against malicious actors with commit access to the repository.
```

Comment:
Linter is reporting `actors` may be insensitive. But I'm not sure what to change it to. Maybe "persons"?

```suggestion
This option is intended to protect your infrastructure from a general Buildkite compromise that might attempt to send arbitrary commands to your agents. It is not designed, nor effective, at protecting against malicious persons with commit access to the repository.
```

Reply:
I think we're fine with actors in this context. I've kicked the linter.

Reply:
```suggestion
This option is intended to protect your infrastructure from a scenario where Buildkite itself gets compromised, and subsequently sends malicious commands to your agents. It is not designed, nor effective at protecting against malicious actors with commit access to the repository.
```